package.json

```
@@ -3,6 +3,9 @@
   "version": "1.0.0",
   "repository": "git@github.com:zendesk/copenhagen_theme.git",
   "scripts": {
+    "start": "rollup -c -w",
+    "build": "rollup -c",
+    "eslint": "eslint src",
```

nitty nit (feel free to ignore 😄)

```suggestion
    "lint": "eslint src",
```